File: javascript/Day_1/Question_08.js
number = 0;

const check = (number) => {
  if (number > 1) {
    console.log(`${number} is positive`);
  } else if (number === 0) {
    console.log(`${number} is zero`);
  } else if (number < 0) {
    console.log(`${number} is negative`);
  } else {
    console.log("number is negative");
  }
};

const result = check(number);

File: dsa/Day_1/Question_08.cpp
#include <iostream>
using namespace std;

void convert(int number){
    if(number > 0){
        cout << number << " is positive number" << endl;
    }else if(number == 0){
        cout << number << " is zero number" << endl;
    }else{
        cout << number << " is negative number" << endl;
    }
}

int main(){
    int number;
    cout << "number: ";
    cin >> number;
    convert(number);

    return 0;
}
